app/models/book.rb

```diff
@@ -1,3 +1,25 @@
 class Book < ApplicationRecord
-  
+  has_many :reviews
+
+	def average
+		self.reviews.average(:rating).round(2)
```

Comment:
we can just use `reviews.average` without the `self` because we are just getting information from the object, we are in an instance of `self` so can just access the attributes